modules/verify-knative-serving-monitoring.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * serverless/monitoring-serverless.adoc
+
+[id="configuring-knative-services-domains_{context}"]
+= Verifying an {product-title} monitoring installation for use with Knative Serving
+
+The OpenShift Serverless operator deploys ServiceMonitor objects during the installation automatically.
+Manual configurations by an administrator are not required, but you can carry out steps to verify that monitoring is installed correctly.
+
```

Comment:
Since the "Verifying an OpenShift Container Platform monitoring installation for use with Knative Serving" subsection is the first in the document, it would make sense to include the Prerequisites also in this subsection, WDYT?

Reply:
Great catch, @mvinkler :star: I've changed the order so this will make more sense.